check-docstyle.sh

```diff
@@ -0,0 +1,72 @@
+directories="f8a_version_comparator tests"
```

Comment:
The file is missing executable bit.

`chmod +x check-docstyle.sh` should do the trick.